diagram.md

```diff
@@ -45,7 +45,7 @@ The following diagram represents a simple Matroska file, comprised of an `EBML D
 
 The Matroska `EBML Schema` defines eight `Top Level Elements`: `SeekHead`, `Info`, `Tracks`, `Chapters`, `Cluster`, `Cues`, `Attachments`, and `Tags`.
 
-The `SeekHead Element` (also known as `MetaSeek`) contains an index of where other `Top Level Elements` of the `Segment` are located in order to let the parser know where the other major parts of the file are. This element isn't technicaly REQUIRED, but without a `SeekHead Element` a `Matroska Parser` would have to search the entire file to find all of the other `Top Level Elements`. This is because Matroska has flexible ordering requirements; for instance, the `Chapters Element` could be stored after the `Cluster Elements`.
+The `SeekHead Element` (also known as `MetaSeek`) contains an index where other `Top Level Elements` of the `Segment`. This index defines where the other major parts of the file are. The `SeekHead Element` is RECOMMENDED. Without a `SeekHead Element`, a Matroska parser would have to search the entire file to find all of the other `Top Level Elements`. This is because Matroska has flexible ordering requirements; for instance, the `Chapters Element` could be stored after the `Cluster Elements`.
```

Comment:
I personally would put `The SeekHead Element is RECOMMENDED.` before the `This index defines where` sentence.